lib/archethic/governance/code/cicd/docker/cicd.ex

```diff
@@ -193,36 +181,66 @@ defmodule Archethic.Governance.Code.CICD.Docker do
     dir = temp_dir("utn-#{address_encoded}-")
     nb_nodes = 5
 
-    compose_prefix = Path.basename(dir)
-    validator_container = "#{compose_prefix}_validator_1"
-    validator_continue = ["ash", "-c", "echo 'yes' > /proc/1/fd/0"]
+    compose_prefix =
+      dir
+      |> Path.basename()
+      |> String.downcase()
 
-    nodes = 1..nb_nodes |> Enum.map(&"#{compose_prefix}_node#{&1}_1")
+    validator_container = "#{compose_prefix}-validator-1"
+
+    nodes = 1..nb_nodes |> Enum.map(&"#{compose_prefix}-node#{&1}-1")
 
     with :ok <- Logger.info("#{dir} Prepare", address: address_encoded),
          :ok <- testnet_prepare(dir, address, version),
          :ok <- Logger.info("#{dir} Start", address: address_encoded),
-         {_, 0} <- testnet_start(dir, nb_nodes),
+         %{cmd: {_, 0}, testnet: _testnet} <- testnet_start(dir, nb_nodes),
```

Comment:
why do you need `testnet` variable as output ?

Reply:
I was doing some tests for multi stage validation, this will be needed for sure that's why I left it there.

Reply:
ok